virtual-desktop/src/app/window-manager/mvd-window-manager/launchbar/launchbar/launchbar.component.ts

```diff
@@ -181,12 +181,23 @@ export class LaunchbarComponent {
       this.applicationManager.spawnApplication(this.propertyWindowPluginDef,this.getAppPropertyInformation(plugin));
     }  
   }
+
+  openStandalone(item: LaunchbarItem): void {
+    const plginType:string = item.plugin.getFramework();
+    if (plginType === 'iframe') {
+      // Still allows IFrames to comprehend URL parameters if address is copy/pasted later
+      window.open(`${location.origin}/ZLUX/plugins/${item.plugin.basePlugin.getIdentifier()}/web/`);
```

Comment:
We'd prefer to have this not hard coded in case our URI changes (like it did from /bootstrap/... last year) but this doesn't happen often and I can't think of a better way to do it atm so this is probably alright

Reply:
Can you try window.ZoweZLUX.uriBroker.pluginResourceUri(item.plugin, '') ??
https://github.com/zowe/zlux-app-manager/blob/master/bootstrap/src/uri/mvd-uri.ts#L105